### ISSUE TITLE
Preset descriptions (encoding features) and benchmark for GTX10**/RTX20**/RTX30** GPUs

### ISSUE THREAD

**Issue by SK-Hardwired** [rigaya/NVEnc#350]
Hi @rigaya  and guys !

Is there any description or map what actual encoding features are used under which preset levels on GTX10**, RTX20** and RTX30** GPUs respectively?

At GTX1060 I always used -u 7 (best, slowest) preset and found with the same preset at my new laptop with RTX30** is slower. Of course I guessed that at RTX30** NVENCC it uses additional features compared to GTX10**, but wonder which ones exactly?

Also it would be nice to have some hint which preset at RTX30** gives the same quality (and what speed) as the GTX10** -U 7 profile setting :)

Thank you!

**Comment by bmcassagne** [comment]
I've struggled to find some good answers on this also.  I can tell you that CQP 25 seems to give good results for 4k with typically very good file size.  Maybe comparable quality to CRF22-24 at slow setting using CPU, with equal or better file size.  It's very subjective obviously.  I'm using adaptive quantization temporal, 2-pass full, and look ahead 32.  For me it's difficult to see what these settings are doing, or rathe if they're working.  But I'm getting 38-50 fps even with NLMEANS denoise (which I'm not really sure is working either).  

Basically my performance is fantastic.  So I'm looking for any possible option to increase quality relative to file size.

**Comment by rigaya** [comment]
> Is there any description or map what actual encoding features are used under which preset levels on GTX10**, RTX20** and RTX30** GPUs respectively?

I haven't seen information about this in the NVENC SDK, so we'll need actually test and check the performance and the quality we'll get. Note that it might depend on driver version also (but not many cases).

RTX30** has HEVC B frame support compared to GTX10**, so quality should have increased, you might want to go for more performance side presets.

**Comment by rigaya** [comment]
I'll close the issue as the topic has been answered.
